pytest/tests/loadtest/contract/src/lib.rs

```diff
@@ -62,4 +62,11 @@ impl Counter {
     pub fn infinite_loop(&self) {
         loop {}
     }
+
+    pub fn write_many(&mut self, how_many: u64) {
+        for i in self.val..self.val + how_many {
+            self.records.insert(&i.to_string(), &"a".to_string());
+        }
+        self.val += how_many;
+    }
```

Comment:
Unrelated to the PR, but I am wondering whether using sdk-based contract for loadtesting makes sense? If I did this from the start, I'd probably write a "raw" contract using just `extern "C"` blocks, like the `near-test-contract` crate does. There are two problems with using SDK:

* conceptual circular dependency: we need to upgrade SDK version, if we add a new feature to nearcore, we'd have to wait for a new SDK release to use it in the loadtest contract
* opaqueness during debugging -- looking at this code, I can't say what the actual runtime behavior here, how many db reads and writes are we issuing here.

Reply:
good points. 

the cases I'm testing here are quite simple/crude - as I cared mostly about 'filling all the shards up' 

we should discuss it during our meeting, how could we improve here.